docs/architecture/ADR-004-interest-and-reserves.md

```diff
@@ -153,25 +126,16 @@ for _, coin := range newReserves {
 
 ### uToken Exchange Rate Impact
 
-In financial terms, accrued interest is split between reserves and lender profits.
-
-The token:uToken exchange rate, which would have previously been calculated by `(total tokens borrowed + module account balance) / uTokens in circulation` for a given token type and associated uToken denomination, must now be computed as `(total tokens borrowed + module account balance - reserved amount) / uTokens in circulation`.
+The token:uToken exchange rate, which would have previously been calculated by `(total tokens borrowed + module account balance) / uTokens in circulation` for a given token and associated uToken, must now be computed as `(total tokens borrowed + module account balance - reserved amount) / uTokens in circulation`.
 
-Also, because Lend, Withdraw, Borrow, and Repay transactions don't affect the token:uToken exchange rate, the exchange rate only needs to be calculated for each token every `InterestEpoch`, and stored so it can be used during transactions.
+Also, because Lend, Withdraw, Borrow, and Repay transactions don't affect the token:uToken exchange rate, it's enough to update the exchange rate during the EndBlocker `AccrueAllInterest`.
```

Comment:
```suggestion
```
We can remove this line altogether since exchange rate is now computed on demand, instead of being stored.

Reply:
I think this is paragraph is meaningful.